src/styles/blocks/card.scss

```diff
@@ -0,0 +1,38 @@
+.card {
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px;
+  box-shadow: 1px 1px 4px gray;
+
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    padding-top: 33px;
+    padding-left: 20px;
+  }
```

Comment:
Consider using `margin` instead of `padding` to center the image within the `.card`. Padding is generally used to create space within an element, while margin is used to create space around an element. If the image is meant to be centered, `margin: auto` is a common technique.